core-cxf/src/main/java/org/jboss/wise/core/client/consumer/WiseCXFConsumerImpl.java

```diff
@@ -263,9 +265,9 @@ else if (target != null && !target.equals("2.2"))
 	  try {
 		if (messageStream != null) {
 		    messageStream.println("Failed to invoke WSDLToJava");
-		    t.printStackTrace(messageStream);
+         log.error("Failed to invoke WSDLToJava", t);
```

Comment:
Similarly to the other PR, this is wrong. If there's a message stream provided to the WiseCXFConsumerImpl, the stacktrace has to be printed to that message stream.